docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -148,6 +148,12 @@ Offset identifier to start pagination from as returned by the `next` field in th
 Numeric offset to start pagination from based on the snapshots matching this request. Using a non-zero value for this parameter is mutually
 exclusive with using the `after` parameter. Defaults to `0`.
 
+`slm_policy`::
```

Comment:
Since we can have multiple polices here, does it make sense to call the parameter `slm_policies` or `slm_policy_patterns`?

Reply:
Sounds good, I went with `slm_policy_filter` now :)